src/main/java/software/sigma/internship/scheduler/ScheduledStatisticDataRetriever.java

```diff
@@ -0,0 +1,35 @@
+package software.sigma.internship.scheduler;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import software.sigma.internship.dto.StatisticDataDto;
+import software.sigma.internship.exception.WebException;
+import software.sigma.internship.mapper.StatisticDataMapper;
+import software.sigma.internship.service.StatisticDataService;
+import software.sigma.internship.service.WarAPIService;
+
+import java.time.LocalDate;
+
+@Component
+@RequiredArgsConstructor
+public class ScheduledStatisticDataRetriever {
+    private final WarAPIService warAPIService;
+    private final StatisticDataService statisticDataService;
+    private final StatisticDataMapper statisticDataMapper;
+
+    @Scheduled(cron = "0 0 10 * * ?")
```

Comment:
In general, it's better to store cron value in application properties. But I thought about it more and decided we have to rework this logic and run this job on an application start-up. It is because our application is hosted on Heroku that will shut down it after 30 minutes of inactivity, so we will have to wake up the application before the job constantly.